common/executable-helpers/src/helpers.rs

```diff
@@ -4,8 +4,10 @@
 use libra_config::config::{LoggerConfig, NodeConfig};
 use libra_logger::prelude::*;
 use libra_types::PeerId;
+use slog::FilterLevel;
 use slog_scope::GlobalLoggerGuard;
 use std::path::Path;
+use std::str::FromStr;
```

Comment:
use std::{path::Path, str::FromStr};

Reply:
Thanks, this is no longer an issue, but I'm surprised 'fmt' or 'xclippy' didn't pick this up...